server.js

```diff
@@ -1348,6 +1348,47 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Bintray version integration
+camp.route(/^\/bintray\/v\/(.+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var path = match[1]; // :subject/:repo/:package (e.g. asciidoctor/maven/asciidoctorj)
+  var format = match[2];
+
+  var options = {
+    method: 'GET',
+    uri: 'https://bintray.com/api/v1/packages/' + path + '/versions/_latest',
+    headers: {
+      Accept: 'application/json'
+    }
+  };
+
+  if (serverSecrets && serverSecrets.bintray_user) {
+    options.auth = {
+      user: serverSecrets.bintray_user,
+      pass: serverSecrets.bintray_apikey
```

Comment:
The use of those credentials seem unfortunately necessary. I have created a BinTray account, but I cannot find where in my profile page I get the API key.

It is similarly unfortunate that the rate limit is so low, we'll see what consequences that has.